lib/msal-common/src/error/InteractionRequiredAuthError.ts

```diff
@@ -40,10 +41,17 @@ export const InteractionRequiredAuthErrorMessage = {
  * Error thrown when user interaction is required.
  */
 export class InteractionRequiredAuthError extends AuthError {
+    timestamp: string;
```

Comment:
1. Please add access modified (public) 
2. Please add comments on all public API.

Reply:
They are public by default - I don't need to include a keyword.

Reply:
Please discuss with JS team about code styles used, as I have seen this used elsewhere in our code. Would like to understand the code style / linting, which governs this.

Adding "public" makes the code more readable, particularly if you don't know what the default is. This is part of the .NET code style btw.

Reply:
[TypeScript repo's style guide](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines) does not make a mention of this, while [Google's typescript style guide](https://google.github.io/styleguide/jsguide.html) says explicit public should be avoided. Our codebase doesn't seem to be consistent, but there's a setting in eslint for enforcing this whichever way we decide to go.